splice_access_api/src/main/java/com/splicemachine/access/configuration/SQLConfiguration.java

```diff
@@ -210,6 +210,9 @@
     public static final String NESTEDLOOPJOIN_BATCH_SIZE = "splice.nestedLoopJoin.batchSize";
     private static final int DEFAULT_NESTEDLOOPJOIN_BATCH_SIZE = 10;
 
+    public static final String CONTROL_EXECUTION_ROWS_LIMIT = "splice.controlExecution.rowsLimit";
+    private static final int DEFAULT_CONTROL_EXECUTION_ROWS_LIMIT = 2000000;
+
```

Comment:
Hmm, my only concern is we have zip for resource management on the hbase side.  I am fine either way, my vote would favor stability (smaller size).